index.html

```diff
@@ -0,0 +1,8 @@
+<html>
+  <head>
+    <META HTTP-EQUIV=Refresh CONTENT="5; URL=http://www.example.org">
+  </head>
+  <body>
+    The url of this site has been changed. Please update your bookmarks!
```

Comment:
maybe "changed to http://......."? I am wondering if apache has a standard page with extra features like a better design, localization, etc.